### DISCUSSION TITLE
Change react-native-firebase Auth SMS message language?

### DISCUSSION THREAD

**Discussion by ilyadynin** [invertase/react-native-firebase#6981] [General]
Is there a way/property to set the language of the verification sms for firebase auth, based on the language?

`            const confirmation = await auth().signInWithPhoneNumber(phoneNumber);
`

**Comment by efstathiosntonas** [comment]
Go to firebase console, select your project, tap Authentication on the left tab, then Templates, at the left bottom you'll see: Template Language

**Comment by mikehardy** [comment]
Yes - this should be possible. I think it is https://rnfirebase.io/reference/auth#setLanguageCode

Have you tried that one?

**Reply by ilyadynin** [reply]
nope, but thank you for your reply, this will probably be the solution :)
